homepage/scripts/scripts.js

```diff
@@ -143,21 +143,23 @@ const CONFIG = {
  * ------------------------------------------------------------
  */
 
+function replaceDotMedia(elem = document) {
+  const resetAttributeBase = (tag, attr) => {
+    elem.querySelectorAll(`${tag}[${attr}^="./media_"]`).forEach((el) => {
```

Comment:
TL:DR; not an issue, but some context

There's a case where you don't want to rewrite the `media_` paths: https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L114-L128

If a navigation menu comes from the federal project, a central content repository that is shared across consumers - 
we want to fetch media from `www.adobe.com/federal/media_hash.png?queryparams=x` since this will point to `https://main--federal--adobecom.hlx.live/media_hash.png?queryparams=x` where those images have been authored.

You'd want to exclude any media with "federal" in the path, or media that doesn't start with `./media_` 

Luckily this already works given that you have the`^=` selector